src/apis.js

```diff
@@ -2,8 +2,8 @@ import axios from "axios";
 
 const gamesApi = axios.create({baseURL: 'https://andrea-nc-games.onrender.com/api/'})
 
-export const getReviews = () => {
-    return gamesApi.get('/reviews').then((res) => {
+export const getReviews = (category, sortBy, orderBy) => {
+    return gamesApi.get('/reviews', {params: {category: category, sort_by: sortBy, order: orderBy}}).then((res) => {
```

Comment:
Great use of axios params to manage the sortby's without loads of if statements!